library/std/src/sys/unix/process/process_unix.rs

```diff
@@ -1,6 +1,7 @@
 use crate::convert::TryInto;
 use crate::fmt;
 use crate::io::{self, Error, ErrorKind};
+use crate::mem::forget;
```

Comment:
This is a minor nit, but could you please import `crate::mem`; and then use `mem::forget`? That seems to be a common pattern in the standard library, and I think it's clearer here.

Reply:
Done